File: Registro de Alunos/app/src/main/java/com/example/artur/registrodealunos/Modelo/Faculdade.java
package com.example.artur.registrodealunos.Modelo;

import java.util.List;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;

@Entity
public class Faculdade {

    @Id Long id;
    private String nome;
    private String email;
    private String contatoPrincipal;
    private List<Aluno> alunos;

    Faculdade() { }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public List<Aluno> getAlunos() {
        return alunos;
    }

    public String getContatoPrincipal() {
        return contatoPrincipal;
    }

    public void setContatoPrincipal(String contatoPrincipal) {
        this.contatoPrincipal = contatoPrincipal;
    }
}

File: RegistroDeAlunosKotlinEJava/app/src/main/java/com/example/artur/registrodealunos/Modelo/Faculdade.kt
package com.example.artur.registrodealunos.Modelo

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id

@Entity
data class Faculdade (var nome: String,
                      var email: String,
                      var contatoPrincipal: String) {
    @Id var id: Long = 0
    var alunos = listOf<Aluno>()

}